index.js

```diff
@@ -295,10 +295,9 @@ function stringToRegexp (path, keys, options) {
 /**
  * Expose a function for taking tokens and returning a RegExp.
  *
- * @param  {Array}  tokens
- * @param  {Array}  keys
- * @param  {Object} options
- * @return {RegExp}
+ * @param  {!Array}  tokens
+ * @param  {Object=} options
+ * @return {!RegExp}
  */
 function tokensToRegExp (tokens, options) {
```

Comment:
The `keys` parameter doesn't actually appear in the argument list so it was removed. The compiler warned about this.